charts/sonarqube-dce/ci/openshift-verifier-values.yaml

```diff
@@ -1,9 +1,12 @@
 OpenShift:
-  enabled: false
+  enabled: true
```

Comment:
Those values will be used by the chart testing tool by default, hence why the cirrus job for the `chart_testing_on_eks` might be failing as we are using openshift values on EKS

Reply:
Fixed!